integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -141,16 +139,19 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
           targetCpu = masterCpu;
           targetMem = masterMem;
           mMasterHostname = offer.getHostname();
-          mMasterLaunched = true;
+          mIsMasterTask = true;
+          mMasterCount++;
+          //&& !mWorkers.contains(offer.getHostname()) add this in cluster environment.
```

Comment:
this comment is confusing, when testing local there will only be one worker ... get rid of this